stl/inc/concepts

```diff
@@ -13,6 +13,7 @@
 #pragma message("The contents of <concepts> are available only with C++20 concepts support.")
 #else // ^^^ !defined(__cpp_lib_concepts) / defined(__cpp_lib_concepts) vvv
 #include <type_traits>
+#include <vcruntime_new.h>

```

Comment:
Is this `(core)` OK?

Reply:
It looks like it but probably worth building a dummy program to check

Reply:
Grepping vcruntime for `pragma comment` shows that:
```c++
#if defined(_M_IX86) && defined(_CRT_LEGACY_X86_FLT_EXCEPTIONS) && !defined(_M_CEE_PURE)
    #pragma comment(lib, "legacy_x86_flt_exceptions")
#endif
```
in `vcruntime.h` (which `yvals_core.h` includes) is the only lib comment. https://github.com/microsoft/STL/wiki/The-Difference-Between-Core-And-Non-Core-Headers says:

> "core" refers to headers that can be #include'd into translation units without generating import declarations forcing linking with the standard libraries.

Together, I believe the implication is that all vcruntime headers are core headers.

Reply:
Sounds good.

Reply:
> It looks like it but probably worth building a dummy program to check
```
C:\Users\Casey\Desktop>type repro.cpp
#include <concepts>
#include <vcruntime_new.h>

int main() {}

C:\Users\Casey\Desktop>cl repro.cpp /nologo /std:c++latest /link /nodefaultlib /ENTRY:main
repro.cpp

C:\Users\Casey\Desktop>cl repro.cpp /nologo /std:c++latest /MD /link /nodefaultlib /ENTRY:main
repro.cpp

C:\Users\Casey\Desktop>
```